### DISCUSSION TITLE
No response when installing on RaspberryPi

### DISCUSSION THREAD

**Discussion by W-Linus** [magicbug/Cloudlog#816] [Q&A] [answered]
I tried to install Cloudlog on my RaspberryPi, but when I clicked install button, it led me to a blank page and then nothing happend.   I have no idea how to solve this problem. Could anyone please tell me how to solve this problem? 

I am using RaspberryPi OS+nginx+MariaDB+PHP7.3.

And I don't know what should be filled into Directory blank when installing. Could anyone please tell me?

VY 73 de BH8PHG.

**Comment by daccle** [comment, marked as answer]
Hi BH8PHG!

- Have you checked that php in general is working? Like having a test.php with phpinfo(); (see: https://www.php.net/manual/en/function.phpinfo.php)
- What are directory and file permissions?
- What is in the log of nginx and php?

I am happy to assist, but I need more information.

Best,
Daniel DC2ZP

**Reply by W-Linus** [reply]
Haha,thanks for your help. I have solved this problem. I just replaced nginx by apache and cloudlog works. Thanks for help.
